RadixWallet/Clients/ROLAClient/ROLAClient+Live.swift

```diff
@@ -36,14 +36,14 @@ extension ROLAClient {
 		return Self(
 			performDappDefinitionVerification: { metadata async throws in
 				_ = try await onLedgerEntitiesClient.getDappMetadata(
-					metadata.dAppDefinitionAddress,
-					validatingWebsite: metadata.origin.url
+					metadata.dappDefinitionAddress,
```

Comment:
The variable name in Rust is `dapp_definition_address`, so it's exported as `dappDefinitionAddress`. Do you know any trick to have it like `dAppDefinitionAddress`? :)

Reply:
On the one hand dApp is the correct style, on the other hand we did decide to generally write it Dapp, while would lead to `dapp...` for instances/enum cases, and `Dapp...` for types. But as you point out, we only seem to follow this convention for types.